src/ColumnSortable/Sortable.php

```diff
@@ -175,7 +175,13 @@ public static function link(array $parameters) //Extending Blade; Blade sends ar
             $anchorClass = 'class="' . $anchorClass . '"';
         }
 
-        return '<a ' . $anchorClass . ' href="'. url(Request::path() . '?' . $queryString) . '"' . '>' . htmlentities($title) . '</a>' . ' ' . '<i class="' . $icon . '"></i>';
+        $clickableIcon = Config::get('columnsortable.clickable_icon', false);
+        $trailingTag = ' ' . '<i class="' . $icon . '"></i>' . '</a>' ;
+        if ($linkOnIcon == false) {
```

Comment:
@Kyslik Martin, I believe you sorted it out, that's supposed to be renamed to `$clickableIcon`.

Well just before I commit, I decided to let the variable names follow your convention in the config file. I renamed the first line and left out the second one. 

I think I used search in my editor and it's set to `exact match`, with a `(` bracket in front of it it's ignored.